up-and-running/platforms/other.md

```diff
@@ -1,3 +1,68 @@
-# Using Other Platforms
+# Other Linux Platforms
 
-## Summary
+npm Enterprise works on most modern versions of Linux:
+
+- Ubuntu 14.04 / 15.10
+- CentOS 7.x
+- Red Hat Enterprise Linux (RHEL) 7.x
+- Debian 7.7
+
+Simply follow the steps listed below:
+
+## Step 1: Install npm and Node.js
+
+Now that you have a server provisioned, let's install npm and Node.js
+
+_On Ubuntu:_
+
+```bash
+curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+sudo apt-get install -y nodejs
+sudo npm i -g npm@latest
+node -v && npm -v
+```
+
+_On Centos/RHEL:_
+
+```bash
+curl -sL https://rpm.nodesource.com/setup_4.x | sudo -E bash -
+sudo yum -y install nodejs
+sudo npm i -g npm@latest
+node -v && npm -v
+```
+
+## Step 2: Install npm Enterprise Itself
+
+```bash
+sudo npm i npme -g --unsafe
+```
+
+To confirm that it worked, type `npme`. If the install succeeded you should see the
+`npme help` screen, which looks like this:
+
+  ![npmo help in terminal](/gitbook/images/npmo-help.png)
```

Comment:
We should update this screenshot to reference `npme` instead. Same for Azure doc.